lib/index.js

```diff
@@ -1,13 +1,37 @@
 /*!
- * Copyright (c) 2018-2023 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
  */
-//import * as bedrock from '@bedrock/core';
-//import assert from 'assert-plus';
-//import {logger} from './logger.js';
+import * as bedrock from '@bedrock/core';
+import {agent} from '@bedrock/https-agent';
+import {httpClient} from '@digitalbazaar/http-client';
 
-// load config defaults
-import './config.js';
+const {config, util: {BedrockError}} = bedrock;
 
-/**
- * @module bedrock-turnstile
- */
+export async function verifyTurnstileToken({token, remoteip}) {
```

Comment:
This is now scoped and the name can be simplified -- importers can rename as needed. We should also use our naming conventions for variable names (camel case) and support switching off on the named client to be used from the config:

```suggestion
export async function verify({token, remoteIp, client = 'default'}) {
```